src/components/capture/create/Spec.tsx

```diff
@@ -0,0 +1,64 @@
+import { Divider, Paper } from '@mui/material';
+import NewCaptureSpecForm from 'components/capture/create/SpecForm';
+import NewCaptureSpecFormHeader from 'components/capture/create/SpecFormHeader';
+import Error from 'components/shared/Error';
+import { Tables } from 'services/supabase';
+import { useQuery, useSelectSingle } from 'supabase-swr';
+
+interface ConnectorTag {
+    connectors: {
+        image_name: string;
+    };
+    id: string;
+    endpoint_spec_schema: string;
+    documentation_url: string;
+}
+
+interface Props {
```

Comment:
Are you leaning towards a generic prop interface name?

Reply:
yeah... makes it easier and you never really export these. Will change others as we have time.